package.json

```diff
@@ -4,35 +4,36 @@
   "version": "1.1.2",
   "author": "Adobe Inc.",
   "engines": {
-    "npm": "^8.0.0",
-    "node": "^16.13 || >=18"
+    "npm": ">= 8.0.0",
+    "node": "^16.13 || ^18 || ^20"
```

Comment:
why not support node 17, 19 and 21? they're LTS and I won't set it as a minimum but they should be fine as well. We could just say
```
"node": ">= 16.13.0"
```

The only package it's not supporting for instance node 21 is our package, none of our dependencies:
```
→ npm i
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@adobe/aio-cli-plugin-aem-rde@1.1.2',
npm WARN EBADENGINE   required: { npm: '>= 8.0.0', node: '^16.13 || ^18 || ^20' },
npm WARN EBADENGINE   current: { node: 'v21.6.2', npm: '10.2.4' }
npm WARN EBADENGINE }

up to date, audited 884 packages in 2s

148 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

Reply:
because uneven node versions are never in LTS and supported for 6 months only. therefore, 17 and 19 are already desupported and 21 will be desupported in june. Also, a transitive library (eslint-plugin-jsdoc) does not support 21 yet. Hence Julian and I decided to stick to even numbers and keep 16 in for now, as customers may not have moved yet, even though 16 is end of life.